<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterValue</class>
 <widget class="QDialog" name="FilterValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxbase">
     <property name="title">
      <string>筛选条件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>性别</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="allSelect">
             <property name="text">
              <string>全选</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="manSelect">
             <property name="text">
              <string>男</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="womanSelect">
             <property name="text">
              <string>女</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>年龄范围</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSpinBox" name="startAgeRange"/>
           </item>
           <item>
            <widget class="QLabel" name="ssss">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>至</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="endAgeRange">
             <property name="value">
              <number>99</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>测试时间</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QDateEdit" name="startTestDate">
           <property name="displayFormat">
            <string>yyyy/M/d</string>
           </property>
           <property name="date">
            <date>
             <year>1980</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="date_lable">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>至</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="endTestDate">
           <property name="date">
            <date>
             <year>2022</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>测试条件</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="3">
          <widget class="QComboBox" name="type">
           <item>
            <property name="text">
             <string>请选择</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>测试距离</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="stroketype">
           <property name="text">
            <string>类别</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="testEnvLabel">
           <property name="text">
            <string>测试环境</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="testEnv">
           <item>
            <property name="text">
             <string>请选择</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QComboBox" name="testDistance">
           <item>
            <property name="text">
             <string>请选择</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="sportLevel">
           <item>
            <property name="text">
             <string>请选择</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="sport">
           <property name="text">
            <string>运动等级</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="strokelabel">
           <property name="text">
            <string>泳姿</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="stroke">
           <item>
            <property name="text">
             <string>请选择</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>所属队伍</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QComboBox" name="team">
           <item>
            <property name="text">
             <string>请选择</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterValue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterValue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
